packages/terra-core-docs/src/terra-dev-site/doc/avatar/AccessibilityGuide.6.doc.mdx

```diff
@@ -0,0 +1,98 @@
+import { Badge } from 'terra-avatar/package.json?dev-site-package';
+import A11yExampleAvatar from './example/Avatar/A11yExampleAvatar';
+
+<Badge />
+
+# Avatar Accessibility Guide
+
+## Why the accessibility of Avatar is important
+
+The Avatar component visually represents a person or facility. Poor or improper implementation of this component can render it inaccessible, which can cause the user inability to understand the avatar.
+
+## Accessibility Considerations
+
+### Content Considerations: What do content creators need to do
+
+- Images with meaningful content must always be assigned proper alternative text. Content creators should provide engineers with relevant text for the component’s `alt` prop that conveys the image's meaning.
+    - If the image conveys no meaning, instruct the engineer that the avatar is purely decorative.
+- If the avatar conveys meaning and is used on top of color other than white, ensure the colors meet the 3:1 non-text contrast ratio between the avatar color and its background color.
+
+### Code Considerations: What do engineers need to do
+
+- Engineers must not override Avatar’s ARIA role.
+    - Engineers must apply an `alt` prop for meaningful images.
+
+<div>
+    <A11yExampleAvatar />
+</div>
+
+```jsx
+const cx = classNames.bind(styles);
+
+const AvatarExample = () => (
+    <Card classNames={cx('card')}>
+        <Card.Body>
+            <Avatar image={exampleAvatarImage} initials="JS" alt="Deep Space" size="1.5em" />
```

Comment:
```suggestion
            <Avatar image={exampleAvatarImage} initials="JS" alt="Avatar" size="1.5em" />
```
Can you change avatar image to this image https://github.com/cerner/terra-core/blob/main/packages/terra-core-docs/src/terra-dev-site/doc/demographics-banner/assets/demographicsCat.jpg 

Can we also have an example created for decorative Avatar with empty alt prop ..??